operations/docker-compose.yml

```diff
@@ -3,8 +3,12 @@ version: "3.3"
 services:
   tf-cli:
     container_name: terraform_state
+    image: ghcr.io/cdcgov/prime-reportstream_tfcli:az2.18.0tf0.14.5
     build:
       context: .
+      args:
+        AZURE_CLI_VERSION: 2.18.0
+        TERRAFORM_VERSION: 0.14.5
```

Comment:
Parameterized the build. Any ideas on how to ensure we only have to update the version numbers in one spot, so the images doesn't also have to be changed?

Reply:
I was hunting for this since I saw the change `ENV` -> `ARG` above :)
I don't know of a clean way (right now) to unify these - to make sure I understand, it's to avoid having to change the image and the args (both in these files), right?

Reply:
Correct, to avoid changing it in two places, eliminating the possibility that we mistag an image with the incorrect version.